Apps/WebClient/src/ClientApp/src/models/dateWrapper.ts

```diff
@@ -60,7 +60,11 @@ export class DateWrapper {
                     });
                     // DST times at 00:00 can be ambigious. Coherse the internal representation to be the day.
                     // 3 Hours bypases the ambiguousness of Move forward and fallback
-                    if (this._internal_date.isInDST) {
+                    if (
```

Comment:
looks like the data is coming with timezone information. In that case I would check that and skip this section.

similar to:
```
if (param.includes("z")) {
                    this._internal_date = DateTime.fromISO(param);
                }

Reply:
@FuriousLlama I have updated the code, can you please check?